### ISSUE TITLE
Need to publish new release of ths plugin

### ISSUE THREAD

**Issue by nandac** [pelican-plugins/pandoc-reader#24]
@justinmayer I would like to publish a new release of this plugin.

I am unable to do this myself and would like you to take me through the process, as right now I am the only maintainer of this project.

The new release is a breaking change and should be version`2.0.0`.

The release notes are as follows:

* BREAKING CHANGE: `PANDOC_DEFAULT_FILES` setting has been renamed to `PANDOC_DEFAULTS_FILES`.
* Modified handling of defaults files to be in sync with Pandoc's handling of these files.

**Comment by justinmayer** [comment]
Hi @nandac. You said:

> I am unable to do this myself…

On the contrary — you are indeed empowered to do this yourself. If you recall, that is precisely how we released version 1.1 of this plugin.

> … and would like you to take me through the process

Certainly. As a reminder, I have configured this repository to automatically publish new releases to PyPI when a `RELEASE.md` file is present. For reference, [you can see how we did this for the 1.1 release](https://github.com/pelican-plugins/pandoc-reader/commit/5d6cbd6d88564f61be02f13c7c7bc86baf36794f#diff-2b1b69303b927a484e02c7fad9fc87d0d3ff0dc22ae1da0ecd0dc935d922a23c).

In short, I suggest making a new pull request containing a `RELEASE.md` file with the following contents:

```
Release type: major

* Rename `PANDOC_DEFAULT_FILES` setting to `PANDOC_DEFAULTS_FILES`
* Handle defaults files in a manner consistent with Pandoc's handling of these files
```

On a side note, a friendly practice would be to deprecate the old setting for one or more releases rather than completely dropping support for it. Perhaps you have already done that (I haven't looked), but I thought I would mention it just in case you hadn't.

**Comment by nandac** [comment]
Thanks, @justinmayer for the refresher. The last release was so long ago I just did not remember.

I will take on your suggestion to not deprecate the setting entirely and make a new PR with a `RELEASE.md` file.

Cheers.
